hw/hw_08_task_02.py

```diff
@@ -0,0 +1,34 @@
+import sqlite3
+from typing import Tuple
+
+
+# flake8: noqa: S608
```

Comment:
We should not disable it. This indicates a real issue.  Do you know about Little Bobby Tables?

Reply:
I had no idea about it. I have googled it just now, the comics is funny :) I'll fix it.